modules/configure-slack.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+//
+// * integration/integrate-with-slack.adoc
+:_module-type: PROCEDURE
+[id="configure-slack_{context}"]
+= Configuring Slack
+
+Start by creating a new Slack App, and get the webhook URL.
+
+.Procedure
+. Create a new Slack App:
++
+[NOTE]
+====
+If you want to use an existing Slack App, go to https://api.slack.com/apps and select an app.
+====
+.. Navigate to https://api.slack.com/apps/new.
+.. Enter the *App Name* and choose a *Development Slack Workspace* to install your app.
+.. Click *Create App*.
+. On the settings page, *Basic Information* section, select *Incoming Webhooks* (under *Add features and functionality*).
+. Turn on the *Activate Incoming Webhooks* toggle.
+. Select *Add New Webhook to Workspace*.
```

Comment:
Playing along at home, I only see "Request to Add New Webhook" presumably because I need admin approval. Do you want to add a prereq or a note in this spot that you must be an admin or receive admin approval to get past the webhook step?